src/main/java/codeu/model/data/User.java

```diff
@@ -19,45 +19,91 @@
 
 /** Class representing a registered user. */
 public class User {
-  private final UUID id;
-  private final String name;
-  private final String password;
-  private final Instant creation;
-
-  /**
-   * Constructs a new User.
-   *
-   * @param id the ID of this User
-   * @param name the username of this User
-   * @param password the password of this User
-   * @param creation the creation time of this User
-   */
-  public User(UUID id, String name, String password, Instant creation) {
-     this.id = id;
-     this.name = name;
-     this.password = password;
-     this.creation = creation;
-  }
-
-  /** Returns the ID of this User. */
-  public UUID getId() {
-    return id;
-  }
-
-  /** Returns the username of this User. */
-  public String getName() {
-    return name;
-  }
-
-  /**
-   * Returns the password of this User.
-   */
-  public String getPassword() {
-    return password;
-  }
-
-  /** Returns the creation time of this User. */
-  public Instant getCreationTime() {
-    return creation;
-  }
+    private final UUID id;
+    private final String name;
+    private final String password;
+    private final Instant creation;
+    private final Integer Age;
+    private final String firstName;
+    private final String lastName;
+    private final String email;
+    private final String phoneNum;
+    private final String bio;
+
+    /**
+     * Constructs a new User.
+     *
+     * @param id
+     *            the ID of this User
+     * @param name
+     *            the username of this User
+     * @param password
+     *            the password of this User
+     * @param creation
+     *            the creation time of this User
+     */
+    public User(UUID id, String name, String password, Instant creation) {
+	this.id = id;
+	this.name = name;
+	this.password = password;
+	this.creation = creation;
+	
+	/** Initializes other information for profile pages. */
+	this.Age = 0;
```

Comment:
These being hard-coded is fine for now, but we should change this when we actually want to be able to edit user information.

That is, at some point:
- Add setter methods for age, first name, last name, e-mail, phone number, bio, etc
- Add another constructor with all these new fields you're adding
- Consider using the [Builder](https://www.javaworld.com/article/2074938/core-java/too-many-parameters-in-java-methods-part-3-builder-pattern.html) pattern when you add another constructor since having a constructor with so many parameters is unwieldy.

Reply:
I like the Builder pattern here too. All tests passing on my branch, I like the use of Integer object so we can have nullability :)
Sorry for late review I need to turn on email notifications when I'm added as a reviewer